meetings/2021-11-17.md

```diff
@@ -0,0 +1,189 @@
+Paths are just public facing phenomena. 
+Those are dynamically generated on the fly. 
+Even if you move container the permissions would stay with it since they are connected to permanent uri.
+
+### Commit previous meeting notes
+
+https://github.com/solid/authorization-panel/pull/272
+Meeting notes merged
+
+
+### Next steps for ACP draft
+
+URL: https://solid.github.io/authorization-panel/acp-specification/
+
+* **Justin**: I haven't been driving spec writing or implementing. 
+I know others were actively involved in doing it. 
+I believe that draft is at a point where it can go through the next steps. Where it would be evaluated for the protocol. 
+I don't think we can accomplish more here before we have that debate. 
+We should also have implementations like CSS which wait for ACP. 
+We have community looking for resolutions for problems that ACP addresses. 
+I don't think we need to determine that nothing else is possible before we evaluate ACP further.
+* ...: Can we start it now or we still need to wait.
+* **Matthieu**: I have identified one major design spec which not makes it orthogonal to solid and other systems as it stands. 
+I want to correct that and release TypeScript implementation of ACP before Jan 1st. 
+After that I would like to move to what you are describing.
+* **Justin**: That's great! It's good to have clarity on the path there. Please share the problem once you document it.
+* **Matthieu**: I will open github issue.
+
+ACTION: Matthieu to open GitHub issue documenting the design flaw.
+
+
+### Constraining clients (apps) should be scoped to a user (social agent)
+
+URL: https://solid.github.io/authorization-panel/acp-specification/#client-matcher & https://solid.github.io/authorization-panel/acp-specification/#policy-satisfaction
+
+* **Elf**: The user should be able to constrain what applications should be able to do on their behalf.
+* Looking at the ACP spec we have
+```turtle
+    ex:matcherA acp:client ex:clientB
+```
+* How would ...
+* **Mattieu**: We can have user/client combination
+*   If you want to restrict clients with specific users you would just pair them together.
+* **Elf**: I will write up an example, and you can tell me if it works out.
+
+
+### Access Matrix, collapsing by resources and agents
+URL: https://github.com/solid/authorization-panel/discussions/203#discussioncomment-610302
+
+* **Elf**: We want to express access Matrix by agent (bob has access to r1, r2...) and resources (resource is accessible for bob, alice...).
+* **Justin**: I think we're talking about several different concepts here. 
+The reason for tackling such concepts in interop is in order to have agents interoperating over data, you need understanding of the data and applications able to express which data is accessible without any knowledge of what data is accessible without understanding of what data is on alice's or bob's pod.
```

Comment:
VERY confusing ultra long sentence!  I hope I rephrased it with original meaning!
```suggestion
The reason for tackling such concepts in interop is in order to have agents interoperating over data. You need understanding of the data and applications in order to express which data is accessible without any knowledge of what data is on alice's or bob's pod.
```